netsim/ansible/tasks/frr/initial-clab.yml

```diff
@@ -14,5 +15,14 @@
   set_fact:
     netlab_mgmt_vrf: False
 
+- name: Load bonding kernel module
```

Comment:
We have a generic "load kernel module X if module Y is active" infrastructure. Use that.